apps/playground/src/main.tsx

```diff
@@ -65,9 +65,8 @@ const ThemeButton = styled.button`
   }
 `;
 
-ReactDOM.render(
+createRoot(document.getElementById('root')!).render(
```

Comment:
eslint didn't say anything about using `!`?

Reply:
seems like not..

Reply:
Oh, for some reason demo app doesn't have eslint config...